code/online_challenges/src/codechef/FLOW006.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
```

Comment:
Move the code to the correct directory and reformat this file using clang-format online. Also remove the using statement.